cmd/samples/recipes/ctxpropagation/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"time"
+
+	"github.com/pborman/uuid"
+	"github.com/samarabbas/cadence-samples/cmd/samples/common"
+	"go.uber.org/cadence/client"
+	"go.uber.org/cadence/worker"
+	"go.uber.org/cadence/workflow"
+)
+
+// This needs to be done as part of a bootstrap step when the process starts.
+// The workers are supposed to be long running.
+func startWorkers(h *common.SampleHelper) {
+	// Configure worker options. Setup two context propagators - one string and
```

Comment:
Comment seems to be misleading.  I only see one ContextPropagator specified in the options below.